### ISSUE TITLE
how to access picorv32_pcpi_mul from software?

### ISSUE THREAD

**Issue by suryacharanp** [YosysHQ/picorv32#242]
Hi, can any one tell me how to access the example pcpi instruction "picorv32_pcpi_mul" from the software. Can I access it using the asm function in c?

**Comment by SyedFahimuddinAlavi** [comment]
did you found the solution of your problem or any problem you faced with PCPI mulitiplication in later steps?

**Comment by suryacharanp** [comment]
No, I was not successful in accessing the in built  pcpi mul from software. It is working when I use a test bench though.
